magefiles/magefile.go

```diff
@@ -12,10 +13,18 @@ import (
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
 
+	toolchainApi "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/md5"
 	gh "github.com/google/go-github/v44/github"
 	"github.com/magefile/mage/sh"
 	"github.com/redhat-appstudio/e2e-tests/pkg/apis/github"
+	client "github.com/redhat-appstudio/e2e-tests/pkg/apis/kubernetes"
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+	apimachineryv1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
ussually we are using metav1 instead of apimachineryv1? Can u change to metav1 please?